pyfrc/mains/cli_deploy.py

```diff
@@ -176,6 +180,26 @@ def run(self, options, robot_class, **static_options):
             )
             return 1
 
+        """
+        Generate a deploy.json
+        """
+        try:
+            repo = git.Repo(search_parent_directories=True)
+            hash = str(repo.git.describe("--tags", "--always"))
+
+            deploy_data = {
+                "git-describe": hash,
+                "build-host": socket.gethostname(),  # os.uname dosent work on systems that use non-unix os
+                "builder": os.getlogin(),
+                "path": robot_path,
+                "build-date": time.strftime("%H:%M:%S", time.localtime()),
```

Comment:
This isn't a date. It should be `datetime.datetime.now().replace(microsecond=0).isoformat()`?

Reply:
should be fixed